en/codelists-guides/codelist-management.rst

```diff
@@ -2,108 +2,115 @@ IATI Codelist Management
+Non-Core Codelists contain conforming values for a particular **integer** version of the IATI Standard, the contents of which are managed by the IATI Technical Team. Non-Core Codelists contain Codes that qualify data, not processes.
+
+These Codelists usually provide lookup information, e.g. Description Type, Geographic Location, Indicator Vocabularies. An addition or modification has a minor impact on the sense and use of the data.
 
-Non-embedded codelists are managed via GitHub at:
+Non-Core Codelists are managed via GitHub at:
 https://github.com/IATI/IATI-Codelists-NonEmbedded
 
--  We aim to make changes to these lists quickly to allow data users to
-   take advantage of the changes immediately. These changes can be made
-   outside of a formal decimal or integer upgrade process.
--  Releases would be tagged in line with standard but the minor changes
-   will be available on the master branch of this repository.
+**Modification Schedule: Non-Embedded Codelists - Non-Core**
 
-Updates to Non-embedded Codelists
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Codes in Non-Core Codelists may be:
 
-Potential changes to Non-embedded Codelists may come to the team via:
+- Added - at any point in time, subject to applicable notification and/or consultation via `IATI Discuss  <https://discuss.iatistandard.org/c/standard-management/non-embedded-codelist-amendments>`__
+- Modified - at any point in time, subject to applicable notification and/or consultation via `IATI Discuss <https://discuss.iatistandard.org/c/standard-management/non-embedded-codelist-amendments>`__
+- Withdrawn - at any point in time, subject to applicable notification and/or consultation via `IATI Discuss <https://discuss.iatistandard.org/c/standard-management/non-embedded-codelist-amendments>`__
+- Removed - at a Major Upgrade, through the applicable consultation process via `IATI Discuss <https://discuss.iatistandard.org/c/standard-management/non-embedded-codelist-amendments>`__
 
--  Personal requests/Support tickets
--  Email notifications
--  Announcements from the source organisations
--  Automatic detection of changes to external sources compared with our own lists.
+Examples of Non-Core Codelists include: Sector Vocabulary, Condition Type, Result Type.
 
-All proposals will be added to the The `Non-embedded Codelist-Amendments Forum <http://support.iatistandard.org/forums/23076626-Non-embedded-Codelist-Amendments>`__
+Non-Embedded Codelists - Replicated
+----------------------
 
-The IATI Technical Team hold regular team meetings once a week. All outstanding proposals will be discussed at that meeting. The IATI Technical Lead (Bill Anderson) has the ultimate responsibility for decisions taken.
+Replicated Codelists contain values for the IATI Standard. The contents are generally managed by a third-party organisation, in exceptional circumstances an additional code can be added. The IATI Technical Team maintain a corresponding copy in a document that conforms to the Codelist Schema.
```

Comment:
> The contents are generally managed by a third-party organisation, in exceptional circumstances an additional code can be added.

This sentence contains a [comma splice](https://en.wikipedia.org/wiki/Comma_splice). This practice seems to be creeping into the documentation a lot of late.